server/tests/models/notifications.spec.js

```diff
@@ -0,0 +1,72 @@
+import chai from "chai";
+import chaiAsPromise from "chai-as-promised";
+import database from "../../models/index";
+import { userData } from "../fixtures/models/userData";
+import articleData from "../fixtures/models/articleData";
+
+chai.use(chaiAsPromise);
+const { Users, Articles, Notifications } = database;
+
+const notificationDependencies = async () => {
+  const createdUser = await Users.create(userData);
+  const userId = createdUser.get("id");
+  const articleTemplate = Object.assign(articleData, { authorId: userId });
+  const articleInstance = await Articles.create(articleTemplate);
+  const articleId = articleInstance.get("id");
+
+  return Promise.resolve({
+    userId,
+    articleId,
+    user: createdUser,
+    article: articleInstance
+  });
+};
+
+beforeEach(async () => {
+  await database.sequelize.sync({ force: true });
+});
+
+describe("Notifications model", () => {
+  it("should create a notification model with valid user and article", async () => {
+    const { userId, articleId, user } = await notificationDependencies();
+    const notification = await Notifications.create({
+      userId,
+      source: articleId,
+      message: "An article is liked"
+    });
+    chai.expect(notification.get("userId")).equals(userId);
```

Comment:
This should be 
`expect(notification.get("userId")).equals(userId);`
not 
`chai.expect(notification.get("userId")).equals(userId);`